ts/features/wallet/component/WalletV2PreviewCards.tsx

```diff
@@ -30,13 +34,24 @@ const WalletV2PreviewCards: React.FunctionComponent<Props> = props => (
       )),
       null
     )}
+    {pot.getOrElse(
```

Comment:
```suggestion
    {pot.toUndefined(
      pot.mapNullable(props.satispayList, s => (
        <>
          {s.map(s => (
            <SatispayWalletPreview key={s.idWallet} />
          ))}
        </>
      ))
    )}
```
could it be a valid refactoring?

Reply:
Yes sure! sounds good!